docs/how-to/h-external-k8s-connection.md

```diff
@@ -0,0 +1,72 @@
+|----------------------------------|-----------|----------------|-------------|---------------------------------------------------------------------------------|
+| kafka-k8s                        | ClusterIP | 10.152.183.77  |             | 65535/TCP                                                                       |
+| kafka-k8s-endpoints              | ClusterIP | None           |             |                                                                                 |
+| kafka-k8s-bootstrap              | NodePort  | 10.152.183.193 |             | 29092:31982/TCP,29093:31262/TCP,29094:30845/TCP,29095:30153/TCP,29096:32601/TCP |
+| kafka-k8s-0-sasl-plaintext-scram | NodePort  | 10.152.183.78  |             | 29092:31211/TCP                                                                 |
+| kafka-k8s-0-sasl-ssl-scram       | NodePort  | 10.152.183.104 |             | 29093:32306/TCP                                                                 |
+| kafka-k8s-1-sasl-plaintext-scram | NodePort  | 10.152.183.125 |             | 29092:31421/TCP                                                                 |
+| kafka-k8s-1-sasl-ssl-scram       | NodePort  | 10.152.183.162 |             | 29093:31710/TCP                                                                 |
+| kafka-k8s-2-sasl-plaintext-scram | NodePort  | 10.152.183.223 |             | 29092:31449/TCP                                                                 |
+| kafka-k8s-2-sasl-ssl-scram       | NodePort  | 10.152.183.117 |             | 29093:30641/TCP                                                                 |
+```
+
+- The `kafka-k8s-bootstrap` NodePort service exposes a selection of ports, with each one being associated with the supported security protocol + authentication mechanism
+    - Supported security protocols include `SASL_PLAINTEXT`, `SASL_SSL` and `SSL`
+    - Supported authentication mechanisms include `SCRAM-SHA-512`, `OAUTHBEARER` and `SSL`
+- The remaining NodePort services each correspond to a specific unit-id, security protocol and authentication mechanism, with fixed Ports and randomly allocated NodePorts
+
+
+> **Note** - The `kafka-k8s` and `kafka-k8s-endpoints` ClusterIP services seen above are created for every Juju application by default as part of the StatefulSet they are associated with. Generally, these services are not relevant to users and can be safely ignored
+
+## Setting up NodePort services
+
+By default, the `kafka-k8s-bootstrap` NodePort service will be created automatically, however the `NodeIP:NodePort` addresses will be initially unreachable.
+
+To connect to Charmed Kafka K8s use Kubernetes-external clients, the aforementioned services must be associated with individual listeners. To achieve this:
+1. Set the charm config option `expose-external=nodeport` on the Kafka K8s application
+    - This instructs each broker unit to create it's own individual NodePort services
```

Comment:
**typo**:
```suggestion
    - This instructs each broker unit to create its own individual NodePort services
```

Reply:
😞 I've been making that mistake since primary school.